package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "league-of-legends-api",
+  "version": "1.0.0",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./config/sequelize.js",
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "lint:eslint": "./node_modules/.bin/eslint --format codeframe .",
+    "lint:pug": "./node_modules/.bin/pug-lint .",
```

Comment:
Got some puglint errors you didn't fix

![image](https://user-images.githubusercontent.com/5775716/84073574-18409000-a99f-11ea-9868-c32ce312a330.png)